src/rtc/h5_prep.py

```diff
@@ -199,107 +233,237 @@ def populate_metadata_group(h5py_obj: h5py.File,
-            ['RTC-S1', 'Product type'],
-        # NOTE: in NISAR, the value has to be in UPPERCASE or lowercase?
+            ['product_type', 'RTC-S1', 'Product type'],
+        'identification/projectName':
+            ['project_name', 'OPERA', 'Project name'],
+        'identification/acquisitionMode':
+            ['acquisition_mode', 'Interferometric Wide (IW)',
+             'Acquisition mode'],
         'identification/lookDirection':
-            ['Right', 'Look direction can be left or right'],
+            ['look_direction', 'right', 'Look direction can be left or right'],
         'identification/orbitPassDirection':
-            [burst_in.orbit_direction, 'Orbit direction can be ascending or descending'],
+            ['orbit_pass_direction', burst_in.orbit_direction.lower(),
+            'Orbit direction can be ascending or descending'],
         # NOTE: using the same date format as `s1_reader.as_datetime()`
-        'identification/zeroDopplerStartTime':
-            [burst_in.sensing_start.strftime('%Y-%m-%dT%H:%M:%S.%f'),
-             'Azimuth start time of product'],
-        'identification/zeroDopplerEndTime':
-            [burst_in.sensing_stop.strftime('%Y-%m-%dT%H:%M:%S.%f'),
-            'Azimuth stop time of product'],
+
         'identification/listOfFrequencies':
-             [['A'], 'List of frequency layers available in the product'],  # TBC
+             [None, ['A'],
+             'List of frequency layers available in the product'],  # TBC
         'identification/isGeocoded':
-            [True, 'Flag to indicate radar geometry or geocoded product'],
+            [None, True,
+            'Flag to indicate radar geometry or geocoded product'],
+        'identification/productLevel':
+            ['product_level', 'L2', 'Product level'],
+        'identification/productID':
+            ['product_id', product_id, 'Product identificator'],
+        # 'identification/productSource':
+        # [platform_id, 'Product source'],
         'identification/isUrgentObservation':
-            [False, 'List of booleans indicating if datatakes are nominal or urgent'],
+            ['is_urgent_observation', False,
+             'List of booleans indicating if datatakes are nominal or urgent'],
         'identification/diagnosticModeFlag':
-            [False, 'Indicates if the radar mode is a diagnostic mode or not: True or False'],
+            ['diagnostic_mode_flag', False,
+             'Indicates if the radar mode is a diagnostic mode or not: True or False'],
         'identification/processingType':
-            [processing_type, 'NOMINAL (or) URGENT (or) CUSTOM (or) UNDEFINED'],
+            ['processing_type', processing_type,
+             'NOMINAL (or) URGENT (or) CUSTOM (or) UNDEFINED'],
+        # datetime format 'YYYY-MM-DD HH:MM:SS'
+        'identification/processingDateTime':
+            ['processing_data_time',
+             datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Comment:
I still see this line not fixed.
```suggestion
            datetime. datetime.now().strftime("%Y-%m-%dT%H:%M:%S.%fZ"),
```

Reply:
I changed the import from `import datetime` to `from datetime import datetime`. Thank you for checking.